include/walkers.h

```diff
@@ -27,7 +27,7 @@ void walk_methods(const T& scope, MethodWalkerFn walker) {
     for (auto vmethod : cls->get_vmethods()) {
       walker(vmethod);
     }
-  };
+  }
```

Comment:
Can you revert these from this PR?  I agree it's kind of weird, but it's a really minor thing and not related to the described change.